server/users.js

```diff
@@ -544,6 +545,16 @@ class User extends Chat.MessageContext {
 		Users.add(this);
 	}
 
+	/**
+	 * @param {UserSettings} [settings]
+	 */
+	applySettings(settings) {
+		if (!settings) return;
+		if ('blockChallenges' in settings) this.blockChallenges = !!settings.blockChallenges;
```

Comment:
Why the `!!`? Won't the object be something like `{blockChallenges: true, ignoreTickets: false}`? In which case it's the same without the `!!`.

Reply:
Because the compiler complains otherwise. I think its stupid as well.